olp-cpp-sdk-core/include/olp/core/thread/TaskScheduler.h

```diff
@@ -77,16 +83,16 @@ class CORE_API TaskScheduler {
 
  protected:
   /**
-   * @brief Abstract enqueue task interface to be implemented by
-   * subclass.
+   * @brief The abstract enqueue task interface that is implemented by
+   * the subclass.
    *
-   * Implement this method in your subclass taking TaskScheduler
-   * as base and provide a custom algorithm for scheduling tasks
+   * Implement this method in the subclass that takes `TaskScheduler`
+   * as a base and provides a custom algorithm for scheduling tasks
    * enqueued by the SDK.
    *
-   * @param[in] func Rvalue reference of the task to be enqueued.
-   * Move this task into your queue. No internal reference is
-   * kept, once called you own the task.
+   * @param[in] func The rvalue reference of the task that should be enqueued.
+   * Move this task into your queue. No internal references are
+   * kept. Once the task is called, you own it.
```

Comment:
```suggestion
   * kept. Once this method is called, you own the task.
```
The ownership of the task is handed over, when `EnqueueTask` is called, not when the task is called.

Reply:
Changed